src/Controller/api/robot.c

```diff
@@ -132,6 +132,17 @@ static void init_remote_control_library() {
   }
 }
 
+static void init_devices(WbRequest *r, int firstTag) {
```

Comment:
```suggestion
static void init_devices_from_tag(WbRequest *r, int firstTag) {
```

This is just an idea.